framework/components/ecl-site-headers/ecl-site-headers.config.js

```diff
@@ -1,19 +1,16 @@
 module.exports = {
   title: 'Site headers',
   label: 'Site headers',
-  tags: ['organism'],
   status: 'ready',
-  variants: [
-    {
-      name: 'default',
-    },
-    {
-      name: 'homepage',
-      context: {
-        global: {
-          language: 'en',
-        },
-      },
-    },
-  ],
+  tags: ['organism'],
+  context: {
+    political: { href: '#', label: 'Commission and its priorities' },
```

Comment:
The thing is, if one day we have to add another link, we will have to refactor the components... Maybe we can abstract it a bit by simply providing a list of links. It implies refactoring the site-switchers component as well